tests/test_cloudformation/test_cloudformation_stack_crud.py

```diff
@@ -243,19 +250,22 @@ def test_get_template_by_name():
 
 @mock_cloudformation_deprecated
 def test_list_stacks():
+    fixed_timestamp = datetime.datetime(2017, 10, 20, 13, 13, 13)
```

Comment:
Would using freezegun like in other tests look nicer when dealing with times?

E.g.
https://github.com/spulec/moto/blob/master/tests/test_dynamodb2/test_dynamodb_table_with_range_key.py#L66
https://github.com/spulec/moto/blob/master/tests/test_sqs/test_sqs.py#L764